app/services/encrypted_attribute.rb

```diff
@@ -38,11 +37,13 @@ def stale?
   private_class_method def self.load_or_build_user_access_key(cost:, key:)
     @_uaks_by_key ||= {}
     uak_lookup = "#{key}:#{cost}"
-    @_uaks_by_key[uak_lookup] ||= UserAccessKey.new(password: key, salt: key, cost: cost)
+    @_uaks_by_key[uak_lookup] ||= Encryption::UserAccessKey.new(
+      password: key, salt: key, cost: cost
+    )
   end
 
   def current_salt
-    user_access_key.cost + OpenSSL::Digest::SHA256.hexdigest(Figaro.env.attribute_encryption_key)
```

Comment:
Hmm, does this change anything? I don't find something that makes up for this if its just a refactor.

Reply:
This is a change to the `#salt` method on `UserAccessKey`. Previously, the `salt` attribute had the cost attribute prepended. I changed it in this PR so it is actually just the salt. This is the only place the salt is used.

Reply:
And for context, this method is called by `#stale?` to compare the user access key to the global user access key to determine if an attribute should be re-encrypted with the new key.